src/hooks/useOktaSession.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react';
+import { useOktaAuth } from '@okta/okta-react';
+
+const useOktaSession = () => {
```

Comment:
Apologies for the comments about comments, but I think a top-level comment on custom hooks like this would be useful! Something like
```suggestion
// This is a custom hook that uses @okta/okta-react to asynchronously call to the Okta API to check to see if the user has a valid Okta session (which the user may have even when they don't have a valid application session)
const useOktaSession = () => {
```